### ISSUE TITLE
Add preselected options to OAS

### ISSUE THREAD

**Issue by adamGallard** [NomisNostab/Topo#110]
It would be great if we could have some preselected options for the OAS e.g. a button that selects all of the Core OAS to stage 3 instead of having to select all nine individually, also maybe buttons to select all of the stages for Each OAS activity.

**Comment by NomisNostab** [comment]
Hi Adam,
Did you know that you can use Ctrl+click to select multiple stages at once.
Also you can type into the select box and the list will be filtered.

**Comment by adamGallard** [comment]
Yes, I knew about these features, but I am going to run the reports quite often as I am integrating them with our own spreadsheet that fills some of the gaps in terrain, e.g. badge ordering and parent/leader emails! so was just looking to have a quicker way or even automate it somehow.
